Misc/NEWS.d/next/Library/2023-02-01-16-41-31.gh-issue-101410.Dt2aQE.rst

```diff
@@ -1,3 +1,2 @@
-Support custom messages for domain errors in the :mod:`math` module
-(:func:`math.sqrt`, :func:`math.log` and :func:`math.atanh` were modified as
-examples).  Patch by Charlie Zhao and Sergey B Kirpichev.
+Support custom messages for domain errors in the :mod:`math` module.
```

Comment:
"Custom messages" implies that the user can specify the message. But this is not so.

Reply:
```suggestion
Added more verbose messages for domain errors in the :mod:`math` module.
Patch by Charlie Zhao and Sergey B Kirpichev.
```
Does this looks better?

Reply:
Suggestion adjusted.